data/polkit.service.in

```diff
@@ -8,7 +8,7 @@ BusName=org.freedesktop.PolicyKit1
 CapabilityBoundingSet=CAP_SETUID CAP_SETGID
 DeviceAllow=/dev/null rw
 DevicePolicy=strict
-ExecStart=@libprivdir@/polkitd --no-debug
+ExecStart=@libprivdir@/polkitd --no-debug --log-level=2
```

Comment:
Does `--no-debug` still make sense if `--log-level` is specified?

Reply:
I asked myself the same question and it seems that it still does make sense, because it reroutes the output from stdout and stderr to /dev/null and that AFAIK affects Glib's library messages too.